docs/reference/ml/df-analytics/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -497,3 +497,55 @@ predictions associated with the class.
 <5> The number of cats in the dataset that are incorrectly classified as dogs.
 <6> The number of documents that are classified as a class that is not listed as 
 a `predicted_class`.
+
+
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/_evaluate
+{
+   "index": "animal_classification",
+   "evaluation": {
+      "classification": { <1>
+         "actual_field": "animal_class", <2>
+         "predicted_field": "ml.animal_class_prediction", <3>
+         "metrics": {
+            "auc_roc" : { <4>
+              "class_name": "dog" <5>
+            }
+         }
+      }
+   }
+}
+--------------------------------------------------
+// TEST[skip:TBD]
+
+<1> The evaluation type.
+<2> The field that contains the ground truth value for the actual animal 
+classification. This is required in order to evaluate results.
+<3> The field that contains the predicted value for animal classification by 
+the {classanalysis}.
+<4> Specifies the metric for the evaluation.
+<5> Specifies the class name that is treated as positive during the evaluation, 
+all the other classes are treated as negative.
+
+
+The API returns the following result:
+
+[source,console-result]
+--------------------------------------------------
+{
+  "classification" : {
+    "auc_roc" : {
+      "score" : 0.8941788639536681, <1>
+      "doc_count" : 4169 <2>
```

Comment:
With https://github.com/elastic/elasticsearch/pull/63302, `doc_count` is no longer reported.
Please remove.

Reply:
@przemekwitek Done.